backend/src/main/java/com/programmers/pcquotation/global/enums/UserType.kt

```diff
@@ -0,0 +1,21 @@
+package com.programmers.pcquotation.global.enums;
+
+enum class UserType(val value: String) {
+	CUSTOMER("customer"),
+	SELLER("seller"),
+	ADMIN("admin"),
+	NOTHING("nothing");
```

Comment:
이 부분은 대문자로 바꾸셨던 것 같습니다!
```suggestion
	CUSTOMER("CUSTOMER"),
	SELLER("SELLER"),
	ADMIN("ADMIN"),
	NOTHING("NOTHING");
```